pkg/controller/modeler/modeler.go

```diff
@@ -173,13 +173,20 @@ func (c *Controller) rebuildAbusePreventionModel(ctx context.Context, realm *dat
 	// potentially less than zero. We need to do the negative check against the
 	// float value before casting to a uint, or else risk overflowing if this
 	// value is negative.
-	nextFloat := math.Ceil(curve(float64(len(ys))))
+	raw := curve(float64(len(ys)))
+	logger.Debugw("computed raw next curve", "next", raw)
```

Comment:
do you want this to stay in?

Reply:
Yes. If it was in, I would have found this a lot sooner because I would have seen `21.00000000000004`